backend/Tasque.BLL/Interfaces/ITaskService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
We have an ICrudEntityService (or something like this) which is done to not create other standard CRUD services

Reply:
I decided to make another one with full implementration coz for this entity we use two data bases and there are some bottlenecks to make it work as expected